specs/git/update_git_username.yaml

```diff
@@ -0,0 +1,14 @@
+---
+name: Change username of git
+command: "git config --global user.name '{{username}}'"
+tags:
+  - git
+description: Sets the URL for a remote git repository to a new URL
```

Comment:
nit: I think this description is out of date. It's describing setting the url of a remote git repository, but the workflow is actually changing the git username

Reply:
fixed!